perlin.js

```diff
@@ -0,0 +1,119 @@
+import * as THREE from 'three';
```

Comment:
Why not just use C++ simplex?

1) Simplex > perlin.
2) We have 5 implementations in our code already (both JS and wasm), so we don't need more.
3) This is the main thing that it's important to do in WASM, because noise generation is almost always the slowest [Pareto/Amdahl component](https://en.wikipedia.org/wiki/Amdahl%27s_law) which we cannot algorithmically optimize

Reply:
https://github.com/webaverse/app/blob/master/simplex-noise.js